paper-toast.html

```diff
@@ -85,31 +84,51 @@
       text: {
         type: String,
         value: ""
+      },
+
+      visible: {
+        type: Boolean,
+        readOnly: true,
+        value: false,
+        observer: '_visibleChanged'
       }
     },
 
+    hostAttributes: {
+      role: 'alert',
+      'aria-live': 'assertive'
+    },
+
+    ready: function() {
+      this.async(function() {
+        this.hide();
+      });
+    },
+
+    /**
+     * Show the toast.
+     * @method show
+     */
+    show: function() {
+      if (PaperToast.currentToast) {
+        PaperToast.currentToast.hide();
+      }
+      PaperToast.currentToast = this;
+      this._setVisible(true);
+      this._ariaAnnounce();
+      this.debounce('hide', this.hide, this.duration);
+    },
+
     /**
      * Hide the toast
      * @return {[type]} [description]
      */
     hide: function() {
-      this.opened = false;
-    },
+      this.setAttribute('aria-live', 'off');
```

Comment:
if you remove this attributes in `hide`, do you need to add them back in `show`?